home/jobs/OMERO-server/config.xml

```diff
@@ -54,6 +54,7 @@ fi
 
 if [ &quot;$PURGE_DATA&quot; = &quot;true&quot; ]; then
     dropdb -h $OMERO_DB_HOST -U $OMERO_DB_USER $OMERO_DB_NAME || echo &quot;First run or already exists&quot;
+    rm -rf $OMERO_DATA_DIR
```

Comment:
yes, this line is in my Build > Execute command of OMERO.server  build and it works in following sense 👍 


  - the devspace was left overnight and the full build (Trigger) was run whilst the "Purge data" was off
  - the images which were in the server were preserved fully, with thumbs and full viewer as verified in the morning
  - previous day in the afternoon, the "Purge data" on was tested as well in this setup and it worked as expected